.circleci/config.yml

```diff
@@ -179,7 +179,8 @@ jobs:
       - image: circleci/python:3.6
     environment:
       TOXENV: py36-ethpm
-      WEB3_INFURA_PROJECT_ID: 4f1a358967c7474aae6f8f4a7698aefc
+      WEB3_INFURA_PROJECT_ID: ae13436d29b34850b640af9c80f80871
+      WEB3_INFURA_API_SECRET: 067ddad6021048f0840a38a4e52eb568
```

Comment:
@kclowes @njgheorghita Tests were failing without `WEB3_INFURA_API_SECRET `. [Here's](https://circleci.com/gh/ethereum/web3.py/79087?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) a failing build. It seems to me that the handshake in websockets is much more strict in this new version(`v8.1`).  I have generated this temporary key pair from my personal account. Please create a new key pair. The infura api is rate limited, so it'd be better to add this environment variables through [circleci's config](https://circleci.com/gh/ethereum/web3.py/79087?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link). Else the tests might fail randomly with rate limit errors if someone starts abusing this key.

Reply:
🤔 that's interesting. It seems like you shouldn't need the secret key unless something changed with that project. I wonder if the project settings associated with the key changed recently? It's not my key so I can't see it.

> it'd be better to add these environment variables through circleci's config. 

We opted to leave the keys in the code (usually with a comment that says please don't abuse) because anyone who contributes has access to the circle env variables anyway and we thought it's better to just put them out there, rather than have someone assume at some point that the CircleCI env vars are actually secret and expose a real vulnerability. If someone abuses we'll just change them out.

Reply:
> Tests were failing without WEB3_INFURA_API_SECRET .

>  It seems like you shouldn't need the secret key unless something changed with that project. 

Yup, nothing's changed with the project settings, so I'm not sure that this was the cause of the error, as it looks like we're not close to hitting the rate limits on that key. For the `4f1....` key, you don't need a secret. I'm guessing it was just a glitchy request with Infura. I just ran the tests with the `4f1...` key and everything seemed to pass just fine.

Reply:
well, I couldn't replicate the error locally on OSX. However, after entering circleCI in debug mode, the secret key is the only thing that fixed the issue. I was getting a 401 unauthorized error. 
Here's a similar [issue](https://github.com/aaugustin/websockets/issues/575) in websockets.

Reply:
I couldn't get circle tests to pass without the secret key either ¯\_(ツ)_/¯. Updated to a throw-away key.